testing/scenario_app/lib/src/platform_view.dart

```diff
@@ -177,7 +177,7 @@ class PlatformViewPartialIntersectionScenario extends Scenario
 
     finishBuilder(
       builder,
-      overlayOffset: const Offset(150, 250),
+      overlayOffset: const Offset(150, 240),
```

Comment:
Because the platform view's frame changed, the old offset won't create an overlay. To make the test valid again, i had to update the offset so that the widget is partially cover the platformview and an overlay is created.

Reply:
I think this is an iOS only test so no android tests are effected by this change.